src/cryptoadvance/specter/templates/wallet/components/wallet_menu.jinja

```diff
@@ -6,12 +6,17 @@
     - active_menuitem: Current active tab. Options: 'transactions', 'addresses', 'receive', 'send', 'settings'.
 	- wallet_alias: The wallet alias (used in the URL navigation).
  #}
-{% macro wallet_menu(active_menuitem, wallet_alias) -%}
+{% macro wallet_menu(active_menuitem, wallet_alias, ext_wallettabs) -%}
 	<nav class="row collapse-on-mobile">
 		{{ wallet_menu_item('history', _('Transactions'), wallet_alias, active_menuitem, isLeft=true, id='btn_transactions') }}
 		{{ wallet_menu_item('addresses', _('Addresses'), wallet_alias, active_menuitem) }}
 		{{ wallet_menu_item('receive', _('Receive'), wallet_alias, active_menuitem, id='btn_receive') }}
 		{{ wallet_menu_item('send', _('Send'), wallet_alias, active_menuitem, id='btn_send') }}
+		{% for ext_id, wallettabs in ext_wallettabs.items() %}
+			{% for wallettab in wallettabs %}
+				{{ wallet_menu_item(wallettab["endpoint"], _(wallettab["title"]), active_menuitem, isRight=false, blueprint=ext_id+"_endpoint") }}
```

Comment:
Are you missing `wallet_alias` here?

Reply:
Yeah, it's missing. Once I added it, I was able to get the tab to render as active.

Reply:
Also, after clicking on my wallet tab, the wallet alias comes in as a request parameter rather than coming in on the request path. So when I click back on one of the other tabs, the wallet alias is no longer part of the request path, and I get the "Wallet history/addresses/receive/send does not exist" alert.

I haven't looked into the cause but I assume that the menu item URLs are being constructed using relative paths.

Reply:
> Also, after clicking on my wallet tab, the wallet alias comes in as a request 
> parameter rather than coming in on the request path.

Not sure why that is. I've added the `wallet_alias` here as well and i've reworked the documentation. I could get rid of the `inject_common_stuff` method and so we only have three modifications left:
* adding the callback_add_-method
* adding the controller-method
* adding the template
So, should work now.